tests/utils_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for english language checker"""
+import unittest
+import libcipher.utils as utils
+
+filepath = '/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt'
```

Comment:
You can find more about File objects in the python [2](https://docs.python.org/2/library/stdtypes.html#bltin-file-objects) documentation and in the python [3](https://docs.python.org/3/glossary.html#term-file-object) documentation as well. 

The idea is to keep opening of the file and storing its content into a `File` object outside of the library to make it independent of where the `dictionary.txt` file is stored.